api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -23,12 +23,12 @@
 import org.pmiops.workbench.cdr.CdrVersionService;
 import org.pmiops.workbench.cdr.dao.CBCriteriaAttributeDao;
 import org.pmiops.workbench.cdr.dao.CBCriteriaDao;
-import org.pmiops.workbench.cdr.model.CBCriteria;
+import org.pmiops.workbench.cdr.model.DbCriteria;
```

Comment:
Why isn't this `DbCBCriteria`? Did you decide it's not cohort-builder-specific, or is that just too much of a mouthful?

Reply:
You should go to Hollywood and shop the rights to this story for a feature film. I think Matt Damon should play CBCriteria.